renovate: always update lvh in tandem with lvh-images

lmb opened the pull request:
lvh-images may need a new version of lvh to boot. Always upgrade lvh when upgrading images.

```release-note
Always update lvh in tandem with lvh-images
```

brlbil commented on .github/renovate.json5:
```diff
@@ -46,7 +46,9 @@
   ],
   "pinDigests": true,
-  "ignorePresets": [":prHourlyLimit2"],
+  "ignorePresets": [
```
Nit, this seems like an auto-correction by the editor.

brlbil commented on .github/renovate.json5:
```diff
@@ -573,4 +578,4 @@
     }
   ]
-}
+}
```
So is this.

lmb committed:
renovate: always update lvh in tandem with lvh-images

lvh-images may need a new version of lvh to boot. Always upgrade lvh
when upgrading images.

Signed-off-by: Lorenz Bauer <lmb@isovalent.com>
